### PR TITLE
Correct the wrong order number in the readme file

### PR DESCRIPTION
**Pull request by PeiHongbing1223**
Refine the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,24 +71,7 @@ Jenkinsfile in SCM 意为将 Jenkinsfile 文件本身作为源代码管理 (Sour
...
-             }
-             steps{
-                  container ('maven') {
-                    sh 'docker tag  $REGISTRY/$DOCKERHUB_NAMESPACE/$APP_NAME:SNAPSHOT-$BRANCH_NAME-$BUILD_NUMBER $REGISTRY/$DOCKERHUB_NAMESPACE/$APP_NAME:latest '
-                    sh 'docker push  $REGISTRY/$DOCKERHUB_NAMESPACE/$APP_NAME:latest '
-                  }
-             }
-          }
-  
-  ```
```
Comment (LinuxSuRen) [inline README.md:89]: I prefer to keep this part.

**Comment by LinuxSuRen** [comment]
In order to make the PR title be more meaningful. I'd like to rewrite it. Please correct me if you have a better one.

/retitle Correct the wrong order number in the readme file

**Review by PeiHongbing1223 — COMMENTED** [review]
I think the title "Correct the wrong order number in the readme file" is very good, you can modify it directly, thank you.

File: README.md (hunk 1 above)
Comment (LinuxSuRen) [reply README.md:89]: oo, I see.

**Pull request merged**